app/config/index.js

```diff
@@ -5,13 +5,15 @@ const storageConfig = require('./storage')
 // Define config schema
 const schema = joi.object({
   env: joi.string().valid('development', 'test', 'production').default('development'),
-  port: joi.number().default(3000)
+  port: joi.number().default(3021),
```

Comment:
Changing to 3021 to be consistent with exposed port and not conflict with others that may be in same container network.